index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="css/style.css" />
+        <title>Apple</title>
+        <meta charset="utf-8" />
+    </head>
+    <body>
+        <main>
+                    <header class="header">
+                            <div id="menu-outer">
+                            <div class="table">
+                                <ul id="horizontal-list">
+                                    <li><a href="#"><div></div></a></li>
+                                    <li><a href="#">Store</a></li>
+                                    <li><a href="#">Mac</a></li>
+                                    <li><a href="#">Ipod</a></li>
+                                    <li><a href="#">Iphone</a></li>
+                                    <li><a href="#">Iped</a></li>
+                                    <li><a href="#">Itunes</a></li>
+                                    <li><a href="#">Support</a></li>
+                                    <li><input type="text"></li>
+                                </ul>
+                            </div>
+                        </div>
+                        
+                    </header>
+                    <div class="text">
+                        <img class="text1" src="img/text1.png" width="980" height="100" alt="What will your verse be?">
+                        <img class="text2" src="img/text2.png" width="980" height="82" alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story.">
+                        <img class="text3" src="img/text3-b.png" width="980" height="32" alt="Watch the “Your Verse” film and explore the stories">
+                    </div>
+        </main>
+        <aside class="contentN">
```

Comment:
For this section please use  instead of ul and li tags please use  <a> tags, with a <div> tag in each `<a>` tag.
Also instead of using image tags put the images as backgrounds of the `<div>` class thanks.

Reply:
First bug -> on my browser it looks ok:
![review1](https://user-images.githubusercontent.com/19622120/62370409-4ad55d00-b532-11e9-8af6-023aefd44abc.jpg)

Secound: what does this mean? use ul or dont use ul
"For this section please use instead of ul and li tags please use tags, with a tag in each <a> tag."
@Ispirett

Reply:
I left a comment on the code addressing your First and second bug comment.

Reply:
Sorry about the confusion I edited the comment.